filesystem.go

```diff
@@ -39,24 +39,36 @@ func (f *filesystem) UUID() string {
 func filesystem2_0(source map[string]interface{}) (*filesystem, error) {
 	fields := schema.Fields{
 		"fstype":      schema.String(),
-		"mount_point": schema.String(),
-		"label":       schema.String(),
+		"mount_point": schema.OneOf(schema.Nil(""), schema.String()),
```

Comment:
Yes, the defaults are applied only after coercing the json blob to the field map.